system_tests/vision.py

```diff
@@ -108,3 +109,104 @@ def test_detect_logos_gcs(self):
         self.assertEqual(len(logos), 1)
         logo = logos[0]
         self._assert_logo(logo)
+
+
+class TestVisionClientFace(unittest.TestCase):
+    def setUp(self):
+        self.to_delete_by_case = []
+
+    def tearDown(self):
+        for value in self.to_delete_by_case:
+            value.delete()
+
+    def _assert_coordinate(self, coordinate):
+        if coordinate is None:
+            return True
+
+        self.assertIn(type(coordinate), [int, float])
+        self.assertGreater(abs(coordinate), 0.0)
+
+    def _assert_likelihood(self, likelihood):
+        from google.cloud.vision.likelihood import Likelihood
+
+        levels = [Likelihood.UNKNOWN, Likelihood.VERY_LIKELY,
+                  Likelihood.UNLIKELY, Likelihood.POSSIBLE, Likelihood.LIKELY,
+                  Likelihood.VERY_UNLIKELY]
+        self.assertIn(likelihood, levels)
+
+    def _assert_landmark(self, landmark):
+        from google.cloud.vision.face import Landmark
+        from google.cloud.vision.face import FaceLandmarkTypes
+
+        self.assertIsInstance(landmark, Landmark)
+
+        valid_landmark_type = getattr(FaceLandmarkTypes,
+                                      landmark.landmark_type, False)
+        if valid_landmark_type:
+            return True
+        return False
```

Comment:
- No testcase calls this method.
- Its name implies it should raise an exception or return silently (the return value doesn't make sense).